zilliqa/src/consensus.rs

```diff
@@ -1341,22 +1364,21 @@ impl Consensus {
         Ok(current.view() == 0 || current.hash() == ancestor.hash())
     }
 
-    fn check_safe_block(&mut self, proposal: &Block) -> Result<bool> {
+    fn check_safe_block(&mut self, proposal: &Block, during_sync: bool) -> Result<bool> {
         let Some(qc_block) = self.get_block(&proposal.qc.block_hash)? else {
             trace!("could not get qc for block: {}", proposal.qc.block_hash);
             return Ok(false);
         };
         // We don't vote on blocks older than our view
         let outdated = proposal.view() < self.view.get_view();
-        let proposal_hash = proposal.hash();
+        let _proposal_hash = proposal.hash();
         match proposal.agg {
             // we check elsewhere that qc is the highest among the qcs in the agg
             Some(_) => match self.block_extends_from(proposal, &qc_block) {
                 Ok(true) => {
-                    let block_hash = proposal_hash;
-                    self.check_and_commit(block_hash)?;
+                    self.check_and_commit(proposal)?;
                     trace!("check block aggregate is outdated? {}", outdated);
```

Comment:
Very minor but I suspect this trace might be misleading for synced nodes - could either delete it, or update it with something like
```suggestion
                    trace!("check block aggregate is outdated? {outdated} {}", if during_sync { "but it doesn't matter because we're syncing" } else { "" });
```